cypress/e2e/CMS/$InfographicCreation.feature

```diff
@@ -0,0 +1,286 @@
+        And browser waits
+        And user clicks on "Select Infographic" button to select media
+        And browser waits
+        And user select "Infographic Display: Article: Medium" from the "Display as" dropdown
+        And browser waits
+        And user selects "center" radio button under "Align"
+        And user clicks on "Embed" button to select infographic
+        And browser waits
+        And user clicks on "Add Body Section" to add a body section
+        And user enters "Article Heading4" as 4 body section heading
+        And browser waits
+        And browser waits
+        And user fills out 4 "Body" text area with "Infographic Display: Article: Medium,Right:"
+        And user clicks the "Insert Infographic" button 4 in the WYSIWYG editor
+        And user enters "Automated Test Infographic" into media title search box and clicks "Select Infographic"
+        And user selects "Automated Test Infographic" item from the media list
+        And browser waits
+        And user clicks on "Select Infographic" button to select media
+        And browser waits
+        And user select "Infographic Display: Article: Medium" from the "Display as" dropdown
+        And browser waits
+        And user selects "right" radio button under "Align"
+        And user clicks on "Embed" button to select infographic
+        And browser waits
+        And user selects "Published" from "Save as" dropdown
+        Then user saves the content page
+        When user clicks on "Content" tab
+        And user clicks on "Add content" action button
+        And user clicks on "Home and Landing" content type
+        Then page title is "Home and Landing"
+        When user selects test site section
+        And user fills out the following fields
+            | fieldLabel               | value                                                                        | field_name                     |
+            | Pretty URL               | home-and-landing-page-to-test-infographic                                    | field_pretty_url               |
+            | Page Title               | Automated Test Home and Landing page to test Infographic                     | title                          |
+            | Browser Title            | Automated Test Home and Landing page to test Infographic - Browser Title     | field_browser_title            |
+            | Card Title               | Automated Test Home and Landing page to test Infographic - Card Title        | field_card_title               |
+            | Meta Description         | Automated Test Home and Landing page to test Infographic - Meta Description  | field_page_description         |
+            | Feature Card Description | Automated Test Home and Landing page to test Infographic - Feature Card Desc | field_feature_card_description |
+        And user enters "Home and Landing page List Description" into "List Description" text field
```

Comment:
so for the home and landing you only creating it to test infographic, so only fill out required fields ( like no need to add all the banner images and taglines and even list description field
so you can remove lines 169 to 185

Reply:
done and entered ticket #447 to handle banner image in Home & Landing page